.github/workflows/secrets-sdk.yml

```diff
@@ -78,6 +78,7 @@ jobs:
       - name: Build wheels
         uses: PyO3/maturin-action@v1
         with:
+          maturin-version: v1.9.4  # https://github.com/PyO3/maturin/issues/2767
```

Comment:
Regarding the comment: this issue was closed on Oct 8th and recent commits show a newer version in use for the action (https://github.com/PyO3/maturin-action/commit/a1726c4d35dd82abd4b357c99ddb0bf67eb3b520) 🤔 do we still need to lock to 1.9.4?